app/views/topics/show.html.erb

```diff
@@ -85,7 +85,7 @@
           <% if @topic.child? %>
             <div class="govuk-!-margin-bottom-1">
               <% if @topic.published? %>
-                <%= link_to 'Archive mainstream browse page',
+                <%= link_to 'Archive specialist topic page',
```

Comment:
This has confused me! Was it incorrect on the main branch, ie was it meant to say **Archive specialist sector page** before? ie there was a bug.

Reply:
I think it sneaked in during the swap to GOV.UK Design System